root/static/scripts/guess-case/MB/GuessCase/Handler/Base.js

```diff
@@ -332,7 +332,7 @@ MB.GuessCase.Handler.Base = function (gc) {
    */
   self.doHyphen = function () {
     if (!gc.regexes.HYPHEN) {
-      gc.regexes.HYPHEN = '-';
+      gc.regexes.HYPHEN = /-|‐/;
```

Comment:
If it's a string, matchWordAtIndex appears to perform an exact match, so to be equivalent you'd need
```suggestion
      gc.regexes.HYPHEN = /^[\-‐]$/;
```
but I'm not sure if this would matter in practice

Reply:
Might not, but might as well change it.